FYDP_ModV1/src/version1.java

```diff
@@ -1,14 +1,181 @@
 import ilog.concert.*;
 import ilog.cplex.*;
 
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Iterator;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellType;
+import org.apache.poi.ss.usermodel.CellValue;
+import org.apache.poi.ss.usermodel.FormulaEvaluator;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import static org.apache.poi.ss.usermodel.Row.MissingCellPolicy;
+
 public class version1 {
 
-	public static void main(String[] args) {
+	public static void main(String[] args) throws IOException {
 		// TODO Auto-generated method stub
 		modelConfig();
 	}
 	
-	public static void modelConfig() {
+	public static void modelConfig() throws IOException {
+	//start of excel read in
+		// write your code here
+        //read
+        String excelFilePath = "/Users/mccurdy/Documents/4B/FYPD/excel_read_in/Jan-30-Front-End.xlsx";
+        FileInputStream inputStream = new FileInputStream(new File(excelFilePath));
+        Workbook workbook = new XSSFWorkbook(inputStream);
+        int numberOfSheets = workbook.getNumberOfSheets();
+        ArrayList<String> readData = new ArrayList<String>(0);
+        //number of teachers
+        int n2 = -1;
+        //list of teachers names
+        ArrayList<String> teacherNames = new ArrayList<String>(0);
+        //teacher allocation
+        ArrayList<Double> FTE = new ArrayList<Double>(0);
+        //Name of schedule
+        String schedule_name = "empty";
+
+        //Individual teacher allocation time staple
+        ArrayList<int[]> availableTime = new ArrayList<int[]>();
+        //number of french teachers
+        //index of first french teacher 
+        int frenchTeachlb = -1;
+        //change to first sheet numberOfSheets to 1
+        for (int i = 0; i < 1; i++) {
```

Comment:
does this need to be written as a loop if its just going once?

Reply:
I think we should also change the name of this variable so its more clear

Reply:
My assumption that this would loop over each sheet. However, I can change it if other people are access their sheets alternatively